ydb/core/kqp/federated_query/kqp_federated_query_helpers.cpp

```diff
@@ -305,20 +305,20 @@ namespace NKikimr::NKqp {
         }
         std::shared_ptr<NYdb::ICredentialsProviderFactory> credentialsProviderFactory = NYql::CreateCredentialsProviderFactoryForStructuredToken(nullptr, structuredTokenJson, false);
         auto driver = federatedQuerySetup->Driver;
-        
+
         NYdb::TCommonClientSettings opts;
         opts
             .DiscoveryEndpoint(endpoint)
             .Database(database)
             .SslCredentials(NYdb::TSslCredentials(useTls))
             .CredentialsProviderFactory(credentialsProviderFactory);
-        auto schemeClient = NYdb::NScheme::TSchemeClient(*driver, opts);
-        return schemeClient.DescribePath(path)
-            .Apply([actorSystem = NActors::TActivationContext::ActorSystem()](const NThreading::TFuture<NYdb::NScheme::TDescribePathResult>& result) {
+        auto schemeClient = std::make_shared<NYdb::NScheme::TSchemeClient>(*driver, opts);
+        return schemeClient->DescribePath(path)
+            .Apply([actorSystem = NActors::TActivationContext::ActorSystem(), p=path, sc = schemeClient](const NThreading::TFuture<NYdb::NScheme::TDescribePathResult>& result) {
                 auto describePathResult = result.GetValue();
                 if (!describePathResult.IsSuccess()) {
-                    LOG_WARN_S(*actorSystem, NKikimrServices::KQP_GATEWAY, "DescribePath failed, " << describePathResult.GetIssues().ToString());
-                    return NThreading::MakeFuture<TGetSchemeEntryResult>(Nothing()); 
+                    LOG_WARN_S(*actorSystem, NKikimrServices::KQP_GATEWAY, "DescribePath failed, path " << p << ", " << describePathResult.GetIssues().ToString());
```

Comment:
```suggestion
                    LOG_WARN_S(*actorSystem, NKikimrServices::KQP_GATEWAY, "Describe path '" << p << "' in external YDB database '" << database << "' with endpoint '" << endpoint << "' failed: " << describePathResult.GetIssues().ToString());
```

Кажется ошибка DescribePath может и в других местах возникать, добавил больше контекста